iplover-app/www/js/data.js

```diff
@@ -259,7 +259,41 @@ iplover.data = (function(){
         localStorage.records = torestore;
         localStorage.user = user;
         return;
-    }
+    };
+	
+	//Returns true if autoSyncOn was called, then turns off autosync.
+	var autoSyncOff = function(){
+		if((localStorage.syncImmediately !== undefined) && JSON.parse(localStorage.syncImmediately)){
+			localStorage.syncImmediately = false;
+			return true;
+		} else{
+			return false;
+		}
+	};
+	
+	//If called, it forces sync when the synchronize page is next loaded.
+	var autoSyncOn = function(){
+		localStorage.syncImmediately = true;
+		return true;
+	};
+	
+	//If the first time called, returns now. 
+	//Otherwise, returns the time set in delaySync.
+//	var timeSincePopup = function(){
```

Comment:
Is this commented out for a reason? Also prefer to just delete commented out code.